tests/test.pipelines.yml

```diff
@@ -1,33 +1,83 @@
-- name: single-end
+- name: single.end
```

Comment:
Why? :exploding_head:

Reply:
I thought I'd match the names of the test files in `tests/data/`